### PR TITLE
Support --arc-version parameter

### LINKED ISSUE #222: add arc version as a parameter to migration.dao

### PR DESCRIPTION
**Pull request by ben-kaufman**
Fix #222

### CODE REVIEW

File: prepare-release.sh [hunk 1]
```diff
@@ -10,6 +10,8 @@ rm -r node_modules
 rm package-lock.json
 npm install
 npm ci
+# Edit default Arc version in migrate.js file
+read -n 1 -s -r -p $'Please edit default Arc version in migrate.js file to the latest version.\nPress any key to continue release.\n'
```
Comment (orenyodfat) [inline prepare-release.sh]: why is that needed ? the version can be fetched from latest

Comment (ben-kaufman) [reply prepare-release.sh]: I initially thought about using latest but this can cause issues when new Arc is released with changes not yet supported by the migration.

File: migrate.js [hunk 2]
```diff
@@ -24,6 +24,7 @@ async function migrate (opts) {
 }
 
 const defaults = {
+  arcVersion: '0.0.1-rc.32',
```
Comment (orenyodfat) [inline migrate.js]: how about taking the default from the installed packaged (`node_module/...`)

Comment (ben-kaufman) [reply migrate.js]: This is also an issue as suppose you just used an older Arc, then it will immediately become the new default instead of the latest.

Comment (orenyodfat) [reply migrate.js]: though you mention is specifically at https://github.com/daostack/migration/blob/master/package.json#L46

Comment (ben-kaufman) [reply migrate.js]: But it changes there as well when you're running the script with a new version.

**Comment by orenyodfat** [comment]
only contracts/0.0.1-rc.16/ ? how about other versions ?
